cmd/fyne/internal/templates/Makefile

```diff
@@ -0,0 +1,14 @@
+# If PREFIX isn't provided, we check for /usr/local and use that if it exists.
+# Otherwice we fall back to using /usr.
+
+LOCAL != test -d /usr/local && echo -n "/local" || echo -n ""
```

Comment:
It ocurred to me that this should probavbly also respect the DESTDIR prefix...

Reply:
We do already? If you look at the install part, we already respect that I think.

Reply:
Literally in this line. We are checking the existence of a target prefix outside of the DESTDIR context

Reply:
Sorry, but I am not quite sure I follow.

Reply:
We will write to $DESTDIR$PREFIX/bin...
to find out PREFIX we check /usr/local vs /usr.
But I think this should be $DESTDIR/usr/local vs $DESTDIR/usr surely?

Reply:
Yeah absolutely you are right. Sorry that I didn't get that initially. Makes a lot of sense.

Reply:
I just implemented the change and verified that local install works and I also set destdir and tested that it worked correctly.